cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -249,12 +248,12 @@ static void usage(boolean error) {
     if (OSDetector.isWindows()) {
       out.println("  cdap.bat [options]");
     } else {
-      out.println("  ./cdap.sh [options]");
+      out.println("  cdap.sh [options]");
     }
     out.println("");
     out.println("Additional options:");
-    out.println("  --web-app-path  Path to Webapp");
-    out.println("  --help          To print this message");
+    out.println("  --ui-path  Path to UI");
```

Comment:
I thought it would be useful for later, but now that I think again, it's better to remove it now.